packages/dashboard/src/components/widgets/primitives/input/useUpdateAssetPropertyValue.ts

```diff
@@ -0,0 +1,38 @@
+import { BatchPutAssetPropertyValueCommand, BatchPutAssetPropertyValueResponse } from '@aws-sdk/client-iotsitewise';
+import { useContext } from 'react';
+import { ClientContext } from '~/components/dashboard/clientContext';
+
+const useUpdateAssetPropertyValue = () => {
+  const client = useContext(ClientContext);
+
+  return (
+    assetId: string,
+    propertyId: string,
```

Comment:
better to use named params so that the function is called as ({ assetId: 'windmi', ...}) instead of ('some string', 'a', 'b') which can make it easy to have the wrong params.

also in the future, this needs to support call via property alias. with your pattern, that would look like (undefined, undefined, propertyAlias) rather than just ({ propertAlias })